src/App.js

```diff
@@ -1,26 +1,17 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.scss";
+import React from 'react';
+import './App.scss';
+import data from './data/dummy.json';
 
-function App() {
+const App = () => {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <ol>
+        {data.map((question, id) => {
+          return <li key={id}>{question.text}</li>;
```

Comment:
The React documentation (https://reactjs.org/docs/lists-and-keys.html#keys) says:

> We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state

Here, the order of questions probably won't change but it's still good practise to not use the index. Each question in the JSON file has an associated id — let's use that for the key